src/components/Core/style.scss

```diff
@@ -2,7 +2,21 @@
 @tailwind components;
 @tailwind utilities;
 
-@import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200');
+@font-face {
+  font-family: 'Material Symbols Rounded';
+  font-style: normal;
+  font-weight: 400;
+  src: url('./fonts/icons.woff2') format('woff2');
```

Comment:
Is it possible to download the font files as a npm dependency? Might be better in case there are some updates to them

Reply:
I'm not sure how to do that

Reply:
It seems there is a 3rd party npm package for that - https://github.com/marella/material-icons, however it doesnt seem like its widely used. I think we can keep it like this for now